README.md

```diff
@@ -1,13 +1,35 @@
 # Shopgate Connect - Extension Boilerplate
 
-This boilerplate contains all important folders and files which are required for an extension.
-
-## Steps to start with the boilerplate
-
-- Replace the `id` property in the file `extension-config.json` with the id of your extension
-- Replace the `version` property in the file `extension-config.json` with version of your wish
-- If you don't need a pipeline, remove the `awesomeOrganization.awesomePipeline.v1.json` in the `pipelines` folder
-- If you want to use a pipelines, modify the `awesomeOrganization.awesomePipeline.v1.json` in the `pipelines` folder and rename it to the new name 
+Extension that will take passed parameters to create a link that uses App Command - openPageExtern
+
+## About Widget Configuration
+The widget can display either a link in text or a image link. There are additional optional configuration for the styling of either the image or text link.
+
+- `link`: the url to be opened in an external browser
+####Use one of the following two configurations.
+- `imgSource`: the source url for your image link
+- `linkText`: the text to display for your text link
+####The following configurations are optional
+- `imageHeight`: the specified height for your image
```

Comment:
we should mention that here all possible css values for height and width are allowed. So %, rem, 'auto' etc

Reply:
Updated